### PR TITLE
fix: lead organisation, additional organisation, contacts following spec

### PR DESCRIPTION
**Pull request by mswertz**
What are the main changes you did:
- changes to display of organisations and contributors in resources detail page

how to test:
- use the test plan

todo:
- [ ] added/updated testplan <= @hslh can you do this?

### CODE REVIEW

**Review by hslh — COMMENTED** [review]
Are you also implementing this point: "Distinguish the list of countries from the list of roles, as they currently have the same font and size." in this PR?

**Comment by hslh** [comment]
>     * [ ]  added/updated testplan <= @hslh can you do this?

Will do this with all the other test plan changes, just not in this PR.

File: apps/nuxt3-ssr/components/ContactCard.vue [hunk 1]
```diff
@@ -33,6 +33,7 @@ defineProps<{
         <span v-if="contact?.prefix"> {{ contact?.prefix }}&nbsp;</span>
         <span v-if="contact?.lastName"> {{ contact?.lastName }} </span>
       </span>
+      <span v-if="contact.organisation">{{ contact.organisation?.name }}</span>
```
[the remark this answers is not in the document]
Comment (mswertz) [reply apps/nuxt3-ssr/components/ContactCard.vue:36]: name is required

File: apps/nuxt3-ssr/components/OrganisationCard.vue [hunk 2]
```diff
@@ -34,23 +30,18 @@ defineProps<{
...
       </div>
 
-      <div v-if="organisation.country" class="mt-3">
-        <p>
-          <i>{{ organisation.country.map((r) => r.name).join(", ") }}</i>
-        </p>
-      </div>
-
-      <img
+      <!-- decided we hide logo for now but will bring back later <img
```
[the remark this answers is not in the document]
Comment (mswertz) [reply apps/nuxt3-ssr/components/OrganisationCard.vue:40]: why? this is a valid todo

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/[resourceType]/[resource]/index.vue [hunk 3]
```diff
@@ -498,7 +498,23 @@ if (route.params.catalogue) {
   crumbs["Resources"] = `/${route.params.schema}/ssr-catalogue/all/resource`;
 }
 
-const contributors = computed(() => resource.value.peopleInvolved);
+const contributors = computed(() =>
+  resource.value.peopleInvolved.sort((a, b) => {
+    const mina = a.role?.length
```
[the remark this answers is not in the document]
Comment (mswertz) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/[resourceType]/[resource]/index.vue:503]: renamed variables

**Comment by mswertz** [comment]
> i would suggest to keep it as before

no

**Comment by mswertz** [comment]
> Are you also implementing this point: "Distinguish the list of countries from the list of roles, as they currently have the same font and size." in this PR?

I tried it didn't look very good. Maybe we can sit together and play with it

File: apps/nuxt3-ssr/components/OrganisationCard.vue [hunk 4]
```diff
@@ -11,20 +11,16 @@ defineProps<{
...
-      <span class="font-bold block">
+    <div class="flex items-start flex-col h-full">
+      <span class="block">
         <span class="font-bold" v-if="organisation?.name">
           {{ organisation?.name }}&nbsp;<template v-if="organisation.acronym"
             >({{ organisation.acronym }})</template
-          >
-        </span>
+          > </span
+        ><br />
```
[the remark this answers is not in the document]
Comment (mswertz) [reply apps/nuxt3-ssr/components/OrganisationCard.vue]: indeed, done

**Review by hslh — APPROVED** [review]

**Pull request merged**